plugins/outputs/cloudwatch/convert_otel_test.go

```diff
@@ -1000,20 +1001,28 @@ func TestConvertOtelExponentialHistogram(t *testing.T) {
 		},
 	}
 
+	// Since some of the values could be zero, we can't use InEpsilon (requires non-zero values to form relative error)
+	// We calculate the relative error directly and then use InDelta instead
+	const episilon = 0.0001
+
 	for _, tc := range testCases {
 		t.Run(tc.name, func(_ *testing.T) {
 			dps := convertOtelExponentialHistogramDataPoints(tc.histogramDPS, "foo", "none", 1, entity)
 
 			assert.Equal(t, 1, len(dps))
 			for i, expectedDP := range tc.expected {
-				assert.Equal(t, dps[i].distribution.Maximum(), *expectedDP.StatisticValues.Maximum, "datapoint Maximum mismatch at index %d", i)
-				assert.Equal(t, dps[i].distribution.Minimum(), *expectedDP.StatisticValues.Minimum, "datapoint Minimum mismatch at index %d", i)
-				assert.Equal(t, dps[i].distribution.Sum(), *expectedDP.StatisticValues.Sum, "datapoint Sum mismatch at index %d", i)
+				assert.InDelta(t, *expectedDP.StatisticValues.Maximum, dps[i].distribution.Maximum(), math.Abs(*expectedDP.StatisticValues.Maximum)*episilon, "datapoint Maximum mismatch at index %d", i)
```

Comment:
nit: Are we trying to make the delta dynamic? I thought `InDelta` just expected a fixed epsilon.

https://pkg.go.dev/github.com/stretchr/testify/assert#InDelta

> InDelta asserts that the two numerals are within delta of each other.

```go
assert.InDelta(t, math.Pi, 22/7.0, 0.01)
```

```suggestion
				assert.InDelta(t, *expectedDP.StatisticValues.Maximum, dps[i].distribution.Maximum(), episilon, "datapoint Maximum mismatch at index %d", i)
```

Reply:
Nevermind. Looks like it was intentional.
> // We calculate the relative error directly and then use InDelta instead